statics/js/components/graph-layout.js

```diff
@@ -1094,13 +1100,37 @@ TopologyGraphLayout.prototype = {
                  'analyzer.bandwidth_absolute_alert']
      };
 
+    if (typeof(Storage) !== "undefind") {
+      var cfgValues = {
+      absolute: [this.getLocalValue("bandwidthAbsoluteActive"),
+                 this.getLocalValue("bandwidthAbsoluteWarning"),
+                 this.getLocalValue("bandwidthAbsoluteAlert")],
+      relative: [this.getLocalValue("bandwidthRelativeActive"),
+                 this.getLocalValue("bandwidthRelativeWarning"),
+                 this.getLocalValue("bandwidthRelativeAlert")]
+      };
+    } else {
+      var cfgValues = {
+        absolute: [0, 0, 0],
+        relative: [0, 0, 0]
+      };
+    }
+
     return $.when(
         vm.$getConfigValue('analyzer.bandwidth_update_rate'),
         vm.$getConfigValue('analyzer.bandwidth_source'),
         vm.$getConfigValue('analyzer.bandwidth_threshold'))
       .then(function(period, src, threshold) {
-        b.updatePeriod = period[0] * 1000; // in millisec
-        b.bandwidth_threshold = threshold[0];
+        if (up = self.getLocalValue("bandwidthUpdatePeriod")) {
+          b.updatePeriod = up * 1000; //in millisec
+        } else {
+          b.updatePeriod = period[0] * 1000;
+        }
+        if (localStorage.bandwidthThreshold) {
+          b.bandwidth_threshold = localStorage.bandwidthThreshold;
```

Comment:
There is a bug in the master branch, it should be b.bandwidthThreshold, can you fix it in your patch

Reply:
done